src/models/rfcProps.js

```diff
@@ -55,6 +55,12 @@ const properties = {
 		readableName: t('contacts', 'Website'),
 		icon: 'icon-public'
 	},
+	geo: {
+		multiple: true,
+		readableName: t('contacts', 'Location'),
+		icon: 'icon-address',
+		defaultjCal: [{}, 'URI', 'geo:90.000,0.000']
```

Comment:
Argh, I just realised it is not valid in vcard3.0.
Let's allow to specify multiple versions on a defaultjCal?
```suggestion
		defaultjCal: {
			'3.0': [{}, 'FLOAT', '90.000;0.000'],
			'4.0': [{}, 'URI', 'geo:90.000,0.000']
		}
```